### PR TITLE
ldc 1.2.0-beta1 (devel)

### PR DESCRIPTION
**Pull request by John-Colvin**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by fxcoudert** [comment]
```
/tmp/ldc-20170318-24293-1nqjg9f/ldc-1.2.0-beta1-src/ldc-lts/dmd2/root/port.c:720:72: error: reference to type 'const llvm::fltSemantics' could not bind to an lvalue of type 'const llvm::fltSemantics &()'
    Port::nan = *reinterpret_cast<const double*>(llvm::APFloat::getNaN(llvm::APFloat::IEEEdouble).bitcastToAPInt().getRawData());
                                                                       ^~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

File: Formula/ldc.rb [hunk 1]
```diff
@@ -27,6 +27,22 @@ class Ldc < Formula
     sha256 "313b430f7066f800b0c9f99f57c17dbbec08650b4014058d4b011bac9bf67830" => :yosemite
   end
 
+  devel do
+    url "https://github.com/ldc-developers/ldc/releases/download/v1.2.0-beta1/ldc-1.2.0-beta1-src.tar.gz"
```
Comment (dunn) [inline Formula/ldc.rb]: Homebrew is scanning the version as `1.2.0`, so you'll need an explicit `version` specification.

**Comment by dunn** [comment]
Could you squash down to one commit, too? Thanks!

**Comment by dunn** [comment]
Merged as c7e500aa86eafc1b82c167d24749ae5013179bcd; thanks!

**Pull request closed without merging**